src/config.c

```diff
@@ -33,17 +33,30 @@
 /*----------------------------------------------------------------------------*/
 /*                            File Scoped Variables                           */
 /*----------------------------------------------------------------------------*/
-#ifdef WAN_FAILOVER_SUPPORTED
 pthread_mutex_t config_mut=PTHREAD_MUTEX_INITIALIZER;
-#endif
+
+//For sending cond signal when cloud status is ONLINE
+pthread_mutex_t cloud_status_mut=PTHREAD_MUTEX_INITIALIZER;
```

Comment:
I dont see cloud_status_mut being used

Reply:
It is getting used getter function https://github.com/xmidt-org/parodus/pull/386/files#diff-18680f57f5d32607a3b196afffe0d6ecb250af6c0a994b304b88fc91231250bfR58
and also in high Qos wait in xmidtsend_rbus.c

https://github.com/xmidt-org/parodus/blob/60c37daa186e11822cdb383973576916d3b898da/src/xmidtsend_rbus.c#L399